### DISCUSSION TITLE
Updating between nightly releases

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#2758] [Q&A] [answered]
Dear Sirs,
I am not sure what is the suggested way in order to update an already installed nightly version.
As far as a I have tested, the newer nightly releases doesn't offer upgrade option when I execute them, so, what can I do for this update / upgrade?

Thanks and Best Regards,
Mike Kranidis

**Comment by JReally** [comment]
Not sure whether this is the best solution but:
I simply uninstall the previous version and install the new version
The previous setting (such as the ID server setting) was preserved so I didn't need to enter again any previous settings.

JR

**Reply by mikekgr** [reply]
Thanks a lot. It's good to keep the previous settings. Let's see if someone from the development team will reply about...

**Comment by mikekgr** [comment]
Well I tested the way to uninstall nightly release in order to install a more recent nightly release. I found problem in case that you use your own server instead of rustdesk public servers. After the uninstall / reinstall the "new" installation doesn't have the private rendezvous server so this approach is not recommended.
Developers please comment.
Thanks.

**Comment by JReally** [comment]
What version of the client are you using... in my case it's for win10/win11.
Not sure what I do but any time I uninstall and reinstall the previous settings are definitely there.

**Reply by mikekgr** [reply]
I tried the uninstall / (re)install way to my Windows 11 from 1.2.0 - 06/Jan/2023 to 1.2.0 - 08/Jan/2023 and in this case, the custom / personal rendezvous server lost and the RustDesk client started with the use of public servers. 
Then it was possible to re-enter my rendezvous server I.P. and key and all goes OK I retained my station I.D. and I was able to work as before, hopefully.

**Comment by rustdesk** [comment, marked as answer]
try out `rustdesk.exe --reinstall` or `rustdesk.exe --update`

**Reply by mikekgr** [reply]
Thanks a lot. I will test it in the forthcoming updates and I will report back in case of problems.
